docs/05-ingest.md

```diff
@@ -187,8 +187,24 @@ configuration. Once the job starts (around 2 minutes), you will be able to follo
 This will allow you to monitor your job and view logs from the Spark executors and driver. You can also access previous job Spark UI's from EMR Studio. 
 After your job finishes the application will auto shutdown after 15 minutes. When in the stopped state it takes seconds for the application to start when a new job is received.
 
-The following can be edited in the Sleeper Admin console. With the
-current stack implementation it is not possible to configure these on a job basis. 
+The following can be edited in the Sleeper Admin console. It it also possible to set these on a per job basis by setting `sparkConf`
+
+An example for overriding at the job level is:
+
+```JSON
+{
+  "tableName": "my-table",
+  "files": [
+    "my-bucket/my-files/"
```

Comment:
There should be a '],' between this line and the next one

Reply:
Missing line added